docs/api/odometry.md

```diff
@@ -48,12 +40,17 @@ There are 508.8 ticks per wheel rotation.
 The `wheel_status` topic produces messages of type [irobot_create_msgs/msg/WheelStatus](https://github.com/iRobotEducation/irobot_create_msgs/blob/rolling/msg/WheelVels.msg).
 This topic publishes at 62.5 Hz.
 
-## The `odom` topic
+## Fused Sensor topics
+
+The following topics are computed on-board the Create 3 robot using its raw sensor data.
+They all publish at 20 Hz.
+
+### The `odom` topic
 
 The `odom` topic produces messages of type [nav_msgs/msg/Odometry](https://github.com/ros2/common_interfaces/blob/rolling/nav_msgs/msg/Odometry.msg).
 The Create® 3 robot fuses the reading from its various sensors in order to produce a dead reckoning estimate of its pose on the `odom` topic.
 
-## The `slip_status` topic
+### The `slip_status` topic
```

Comment:
slip_status and stop_status are published at 20Hz?  I thought they were at 62.5, but I might be wrong.